src/cmd_all/Cargo.toml

```diff
@@ -8,11 +8,15 @@ license = { workspace = true }
 repository = { workspace = true }
 
 [features]
+default = ["rw-static-link"]
 rw-static-link = ["workspace-config/rw-static-link"]
 rw-dynamic-link = ["workspace-config/rw-dynamic-link"]
-embedded-deno-udf = ["risingwave_expr/embedded-deno-udf"]
-embedded-python-udf = ["risingwave_expr/embedded-python-udf"]
-default = ["rw-static-link"]
+all-udf = ["external-udf", "wasm-udf", "js-udf", "deno-udf", "python-udf"]
+external-udf = ["risingwave_expr_impl/external-udf"]
+wasm-udf = ["risingwave_expr_impl/wasm-udf"]
+js-udf = ["risingwave_expr_impl/js-udf"]
+deno-udf = ["risingwave_expr_impl/deno-udf"]
+python-udf = ["risingwave_expr_impl/python-udf"]
```

Comment:
My consideration is that each UDF backend brings many dependencies, and most of our developers don't touch UDF at all. So it would be better to skip them in local development. The same thing could be applied to connectors. For example, I almost never use any connector in my development. Those dependencies could be skipped as well to save my disk.

Reply:
BTW, I think it's time to split connector implementations off and design a clear interface for it, as there are now community developers willing to develop new connectors. 🤔

Reply:
> BTW, I think it's time to split connector implementations off

BTW, https://github.com/risingwavelabs/risingwave/pull/12981 🤔

Reply:
Yeah I noticed this PR. It's a little pity it was not merged, but I think it's in the right direction.